ebu_tt_live/documents/ebutt1.py

```diff
@@ -0,0 +1,109 @@
+from ebu_tt_live import bindings
+from ebu_tt_live.bindings import _ebuttlm as ebuttlm
+from ebu_tt_live.documents import SubtitleDocument
+
+
+class EBUTT1ObjectBase(object):
+    message_type_mapping = {}
+
+    def get_xml(self):
+        raise NotImplementedError()
+
+    def get_dom(self):
+        raise NotImplementedError()
+
+    @classmethod
+    def create_from_xml(cls, xml):
+        instance = bindings.CreateFromDocument(
+            xml_text=xml
+        )
+        if isinstance(instance, ebuttlm.message_type):
+            return cls.message_type_mapping[instance.header.type].create_from_raw_binding(instance)
+
+    @classmethod
+    def create_from_raw_binding(cls, **kwargs):
+        raise NotImplementedError()
+
+
+class TimelineUtilMixin(object):
+
+    def add_to_timeline(self, element):
+        pass
+
+
+class EBUTT1Document(TimelineUtilMixin, SubtitleDocument, EBUTT1ObjectBase):
+    """
+    This class wraps the binding object representation of the XML and provides the features the applications in the
+    specification require.
+    """
+
+    # The XML binding holding the content of the document
+    _ebutt1_content = None
+
+    def _cmp_key(self):
+        raise NotImplementedError()
+
+    def __init__(self):
+        self.load_types_for_document()
+        self._ebutt1_content = bindings.tt()
+        self.validate()
+
+    def validate(self):
+        # Run validation
+
+        result = self._ebutt1_content.validateBinding(
+            availability_time=None,
```

Comment:
I don't know what availabilty time is,  I suspect the reason why it is user here is because the EBUTT1 doc is using the `TimingValidationMixin` which has a `self._semantic_dataset['availability_time']`.
If availabilty time is not allowed in an EBUTT1 doc then I could add some logic to only use `self._semantic_dataset['availability_time']` if the document is NOT an EBUTT1 doc.

Reply:
Availability time is the time when a document becomes available for processing, and is needed for resolving a Part 3 document's active begin and end times. It isn't needed at all for processing a Part 1 document.

Reply:
Easy path here is to leave this as is. It could be that one day we want to do clever processing of Part 1 documents with clock times and actually take their availability time into account as a temporal clipping of the beginning of the document.